x-pack/plugins/integration_assistant/server/graphs/log_type_detection/prompts.ts

```diff
@@ -24,6 +24,8 @@ Follow these steps to do this:
   * 'structured': If the log samples have structured message body with key-value pairs then classify it as "name: structured". Look for a flat list of key-value pairs, often separated by some delimiters. Consider variations in formatting, such as quotes around values ("key=value", key="value"), special characters in keys or values, or escape sequences.
   * 'unstructured': If the log samples have unstructured body like a free-form text then classify it as "name: unstructured".
   * 'cef': If the log samples have Common Event Format (CEF) then classify it as "name: cef".
+  * 'leef': If the log samples have Log Event Extended Format (LEEF) then classify it as "name: leef".
```

Comment:
do we need to add these log to the enum: https://github.com/elastic/kibana/blob/main/x-pack/plugins/integration_assistant/common/api/model/common_attributes.schema.yaml#L57 ? 

or no, since we do not plan to support them at this time?

Reply:
Probably good to do that... Will fix